cmd/swarm/flags.go

```diff
@@ -125,6 +125,11 @@ var (
 		Usage:  "ENS API endpoint for a TLD and with contract address, can be repeated, format [tld:][contract-addr@]url",
 		EnvVar: SwarmEnvENSAPI,
 	}
+	RnsAPIFlag = cli.StringFlag{
```

Comment:
why can't we reuse the existing ENS api flags and just rename them to be something like `--resolver-apis` or something like that? I would appreciate if we find a way to generalize the current implementation we have

Reply:
ENS API uses multi resolver, and RNS is operating only on one TLD. Generalising rns with ens could be a larger task then for this PR. Maybe it is better to keep ens and rns resolvers separate as we do not need multi resolver for rns. 

I am not sure that rns resolver should be associated with ens at all? If it can be associated, then maybe `-ens-api rsk:[address]` could be used instead. But given that rns has its own resolvers and client, it makes sense to have a dedicated flag.

Reply:
See [previous comments](https://github.com/ethersphere/swarm/pull/1939#discussion_r348103413).